srcpkgs/python3-binaryornot/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-binaryornot'
+pkgname=python3-binaryornot
+_pkgname=binaryornot
+version=0.4.4
+revision=1
+wrksrc="${_pkgname}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-chardet>=3.0.2"
```

Comment:
please remove the version constraints (similar change should also be done in other packages)